package.json

```diff
@@ -14,6 +15,21 @@
     "lint": "standard",
     "ci": "npm run lint && MOCHA_FILE=$CIRCLE_TEST_REPORTS/junit/test-results.xml nyc mocha --reporter mocha-junit-reporter && nyc check-coverage"
   },
+  "nyc": {
+    "all": true,
+    "reporter": [
+      "html",
+      "text"
+    ],
+    "statements": 86,
+    "lines": 90,
+    "branches": 29,
+    "functions": 88,
+    "exclude": [
+      "coverage",
+      "nodeApps"
```

Comment:
need to add the `test` directory as well here.

Reply:
useful not to exclude test I think, as if you leave it in it shows you tests that are not doing anything.